packages/shared/src/components/Feed.tsx

```diff
@@ -197,6 +193,23 @@ export default function Feed<T>({
     ranking,
   );
 
+  const onPostLinkClick: FeedPostClick = async (
+    post,
+    index,
+    row,
+    column,
+    event,
+    fromReadArticle,
+  ) => {
+    if (!postModalByDefault || fromReadArticle) {
+      await onPostClick(post, index, row, column);
+      return;
+    }
+
+    event.preventDefault();
```

Comment:
It was due to the issue in utilizing the anchor element as the clickable one but since we are replacing it now with a button, we will be removing this part.